app/api/question_routes.py

```diff
@@ -41,33 +41,38 @@ def user_saves():
     questions = Question.query.join(Question.saves).filter(Save.user_id == user_id)
     return {'questions': [question.to_dict_list_page() for question in questions]}
 
-
 @question_routes.route('/<int:question_id>')
 def question_details(question_id):
+    """
+        returns all details of a question
+    """
     question = Question.query.get(question_id)
-    
+
     if not question:
         return {'errors': {'message': 'Question could not be found'}}, 404
-    
+
+    print('question details:', question.to_dict_details())
+
     return question.to_dict_details()
-    
-    
+
+
 @question_routes.route('/', methods=['POST'])
 @login_required
 def create_question():
     """
     Create a new question
     """
-    
+
     form = QuestionForm()
+    form['csrf_token'].data = request.cookies['csrf_token']
     if form.validate_on_submit():
         new_question = Question(
             user_id = current_user.id,
             title = form.title.data,
             details = form.details.data,
             expectation = form.expectation.data
         )
-        
+
         db.session.add(new_question)
         db.session.commit()
         return new_question.to_dict(), 201
```

Comment:
Need to return form errors unindented after this return statement for when form does not validate